aries-hbase/src/main/java/org/apache/aries/PutWorker.java

```diff
@@ -185,7 +189,7 @@ public void run() {
       try {
         mutator = connection.getBufferedMutator(param);
         while (running) {
-          String k = ToyUtils.generateRandomString(10);
+          String k = ToyUtils.generateRandomString(key_length.value());
```

Comment:
Oh my fault... it is not actually about size of the key... I know why you wrote "How long..."

Reply:
Then you should not take my advice.. can change it back, but try to be more descriptive.